litmus-portal/frontend/src/pages/HomePage/style.ts

```diff
@@ -105,19 +122,22 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'row',
     marginTop: theme.spacing(3.75),
     marginBottom: theme.spacing(2),
-    [theme.breakpoints.down('md')]: {
```

Comment:
Kindly use standard and common breakpoints rather than a specific number

Reply:
This again is required as the standard ones did not work correctly. @whitetiger1399

Reply:
The standard ones should work. You may try with other standard breakpoints and set the most suitable one rather than coming up with a specific number that is not as per Litmus UI

Reply:
They are not working due to the design